R/gr_positions.R

```diff
@@ -126,6 +130,7 @@ downstreamOfPerGroup <- function(tx, downstreamOf) {
 #' @param downstreamFrom a vector of integers, for each group in tx, where
 #' is the new start point of first valid exon.
 #' @return a GRangesList of downstream part
+#' @family GRangesPositions
```

Comment:
completely wierd name for a families
ORFHelpers sounds wrong how about simply "ORFs" as family,
how about "GRanges" its just family name in OUR package